container.go

```diff
@@ -140,22 +140,24 @@ func RootedTransactioner() repository.RootedTransactioner {
 			panic(err)
 		}
 
-		var copier repository.Copier
+		var remote repository.Fs
 		if config.HDFS == "" {
-			copier = repository.NewLocalCopier(
-				osfs.New(config.RootRepositoriesDir), config.BucketSize)
+			remote = repository.NewLocalFs(osfs.New(config.RootRepositoriesDir))
 		} else {
-			copier = repository.NewHDFSCopier(
+			remote = repository.NewHDFSFs(
```

Comment:
It literally means HDFS. The reason it ends with `Fs` even if the name contains it already is because `HDFS` is the name of the thing and this is like: `${THING}Fs`.